### PR TITLE
reformatted tracing-api.md and added TOC.

### PR DESCRIPTION
**Pull request by rghetia**
resubmitting #116 after changing email on CLA.

### CODE REVIEW

**Comment by rghetia** [comment]
i signed it

File: specification/tracing-api.md [hunk 1]
```diff
@@ -34,14 +87,17 @@ TODO: How tracer can be constructed? https://github.com/open-telemetry/opentelem
...
+returns the current Span from the current context.
 
 There should be no parameter.
 
 Returns the default `Span` that does nothing and has an invalid `SpanContext` if no
 `Span` is associated with the current context, otherwise the current `Span` from the context.
 
-#### `WithSpan`: enters the scope of code where the given `Span` is in the current context.
+#### WithSpan
+enters the scope of code where the given `Span` is in the current context.
```
Comment (bogdandrutu) [inline specification/tracing-api.md]: Because this is a sentence should we start with upper case letter?

**Comment by SergeyKanzhelev** [comment]
I'm merging. @rghetia when you have a moment - please send a separate PR to uppercase first 
letters in sentences as @bogdandrutu suggested

File: specification/tracing-api.md (hunk 1 above)
Comment (rghetia) [reply specification/tracing-api.md]: yes. I'll send a new PR.

**Pull request merged**